packages/dds-components/src/components/Select/NativeSelect/NativeSelect.tsx

```diff
@@ -45,8 +54,33 @@ export const NativeSelect = ({
   style,
   onKeyDown,
   onMouseDown,
+  clearable,
+  onChange,
   ...rest
 }: NativeSelectProps) => {
+  const { t } = useTranslation();
+  const selectRef = useRef<HTMLSelectElement>(null);
+
+  const [hasValue, setHasValue] = useState(false);
+
+  useEffect(() => {
+    const selectedIndex = selectRef.current?.selectedIndex ?? -1;
+    setHasValue(selectedIndex !== 0);
```

Comment:
Blir dette riktig? Hvis vi ikke har en `selectedIndex` så har vi vel ikke value? 🤔 

```suggestion
    const selectedIndex = selectRef.current?.selectedIndex ?? -1;
    setHasValue(selectedIndex !== -1);
```